src/style/mod.rs

```diff
@@ -132,12 +132,18 @@ impl Default for Position {
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub enum Overflow {
     /// The automatic minimum size of this node as a flexbox/grid item should be based on the size of it's content.
+    /// Content that overflows this node *should* contribute to the scroll region of it's parent.
```

Comment:
Nit: "its" should be used rather than "it's" throughout the docs for this type.

Reply:
Gah. I do this every single time (because my intuition is that both `it's` and `its` ought to have an apostrophe).